Content.Shared/Paint/PaintComponent.cs

```diff
@@ -0,0 +1,55 @@
+using Content.Shared.Chemistry.Reagent;
+using Content.Shared.FixedPoint;
+using Robust.Shared.Prototypes;
```

Comment:
```suggestion
```

Reply:
need this for ProtoId<ReagentPorototype>